### PR TITLE
fixes toy dualsabers gaining damage

### PR DESCRIPTION
**Pull request by Unit2E**
Got me a much cleaner fix for the dualsaber toy problem. Long story short, the toy version of the saber was inheriting the twohanded component, including damage, from the real dualsaber (as the toy is a subtype). Which means that as soon as it was turned on, it would gain the 34 force.

This keeps spinning and flipping intact too, it only fixes the damage.

🆑 Unit2E
fix: Fixes the toy version of the dual esword becoming almost-real when turned on.
/🆑

### CODE REVIEW

File: code/game/objects/items/dualsaber.dm [hunk 1]
```diff
@@ -23,15 +23,17 @@
 	max_integrity = 200
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 70)
 	resistance_flags = FIRE_PROOF
+	var/real = TRUE
 	var/hacked = FALSE
 	var/brightness_on = 6 //TWICE AS BRIGHT AS A REGULAR ESWORD
 	var/list/possible_colors = list("red", "blue", "green", "purple")
 	var/wielded = FALSE // track wielded status on item
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/dualsaber.dm]:
```suggestion
	var/wielded = FALSE // track wielded status on item
/// Damage of the sword when activated
	var/two_hand_force = 34

```

File: code/game/objects/items/dualsaber.dm [hunk 2]
```diff
@@ -23,15 +23,17 @@
...
 
 /obj/item/dualsaber/ComponentInitialize()
 	. = ..()
-	AddComponent(/datum/component/two_handed, force_unwielded=3, force_wielded=34, \
-					wieldsound='sound/weapons/saberon.ogg', unwieldsound='sound/weapons/saberoff.ogg')
+	if(real)
+		AddComponent(/datum/component/two_handed, force_unwielded=3, force_wielded=34, \
+						wieldsound='sound/weapons/saberon.ogg', unwieldsound='sound/weapons/saberoff.ogg')
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/dualsaber.dm]:
```suggestion
	AddComponent(/datum/component/two_handed, force_unwielded=force, force_wielded=two_hand_force, wieldsound='sound/weapons/saberon.ogg', unwieldsound='sound/weapons/saberoff.ogg')
```

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -394,11 +394,14 @@
 	throwforce = 0
 	throw_speed = 3
 	throw_range = 5
+	real = FALSE
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/toys.dm]:
```suggestion
	two_hand_force = 0
```

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -394,11 +394,14 @@
...
 	attack_verb = list("attacked", "struck", "hit")
 
 /obj/item/dualsaber/toy/ComponentInitialize()
 	. = ..()
-	AddComponent(/datum/component/two_handed, wieldsound='sound/weapons/saberon.ogg', unwieldsound='sound/weapons/saberoff.ogg')
+	if(!real)
+		AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=0, \
+					wieldsound='sound/weapons/saberon.ogg', unwieldsound='sound/weapons/saberoff.ogg')
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/toys.dm]: you can delete 400-404 now i'm pretty sure.

**Comment by Unit2E** [comment]
Thanks for looking at it! Did the stuff

**Pull request merged**